R/fullOPEN-PROM.R

```diff
@@ -148,7 +148,35 @@ fullOPEN_PROM <- function() {
               sep = ",",
               col.names = FALSE,
               append = TRUE)
-
+  
+  x <- calcOutput("IDataElecSteamGen", aggregate = FALSE)
```

Comment:
why not `aggregate = TRUE`? capacities are extensive variables

Reply:
@giannou We discussed this with @Fsioutas , and he made another commit where `aggregate = TRUE`. Review the changes when possible.